runtime/src/accounts_index.rs

```diff
@@ -3616,11 +3616,7 @@ pub mod tests {
     }
 
     fn account_maps_len_expensive<T: IndexValue>(index: &AccountsIndex<T>) -> usize {
-        index
-            .account_maps
-            .iter()
-            .map(|bin_map| bin_map.read().unwrap().len())
-            .sum()
+        index.storage.storage.stats.total_count()
```

Comment:
How exact does this return value need to be? I.e. can callers tolerate different values between the old impl and the new impl?

Mostly wondering due to the `Ordering::Relaxed` on the underlying `count`, which may result in short term visibility differences. I'm assuming that since this is via stats that it should be fine, but wanted to confirm.

Minor: Also, maybe this function could be renamed if it is no longer expensive to calculate? Or, maybe both versions can co-exist? 

And if the values _do_ need to be exactly the same between the two different impls, could have a `debug_assert!()` that performs the old impl _in addition_ to the new one to confirm they return the same value.

Reply:
This function is only implemented in tests. The main thing is that it was previously calculated using just the in_mem map, which is not correct. I am imagining Relaxed will be fine.

Reply:
Got it, makes sense. This is what I get for not looking at the broader context around this function. Thanks for the explanation.